std/outbuffer.d

```diff
@@ -368,18 +374,24 @@ class OutBuffer
         }
 }
 
+///
 @safe unittest
 {
     import std.string : cmp;
 
     OutBuffer buf = new OutBuffer();
 
     assert(buf.offset == 0);
-    buf.write("hello"[]);
```

Comment:
I assume this is a relict from old times? In this case, it's even sliced internally:

```d
void write(const(ubyte)[] bytes)
{
    reserve(bytes.length);
    data[offset .. offset + bytes.length] = bytes[];
    offset += bytes.length;
}
...
void write(in char[] s) @trusted             /// ditto
{
    write(cast(ubyte[]) s);
}
```